VMEncryption/main/DiskUtil.py

```diff
@@ -397,44 +397,44 @@ def get_crypt_items(self):
-                crypt_item.mapper_name = "osencrypt"
+        if encryption_status["os"] == "Encrypted" and not rootfs_crypt_item_found:
+            crypt_item = CryptItem()
+            crypt_item.mapper_name = "osencrypt"
 
-                proc_comm = ProcessCommunicator()
-                grep_result = self.command_executor.ExecuteInBash("cryptsetup status osencrypt | grep device:", communicator=proc_comm)
+            proc_comm = ProcessCommunicator()
+            grep_result = self.command_executor.ExecuteInBash("cryptsetup status osencrypt | grep device:", communicator=proc_comm)
 
-                if grep_result == 0:
-                    crypt_item.dev_path = proc_comm.stdout.strip().split()[1]
-                else:
-                    proc_comm = ProcessCommunicator()
-                    self.command_executor.Execute("dmsetup table --target crypt", communicator=proc_comm)
+            if grep_result == 0:
+                crypt_item.dev_path = proc_comm.stdout.strip().split()[1]
+            else:
+                proc_comm = ProcessCommunicator()
+                self.command_executor.Execute("dmsetup table --target crypt", communicator=proc_comm)
 
-                    for line in proc_comm.stdout.splitlines():
-                        if 'osencrypt' in line:
-                            majmin = filter(lambda p: re.match(r'\d+:\d+', p), line.split())[0]
-                            src_device = filter(lambda d: d.majmin == majmin, self.get_device_items(None))[0]
-                            crypt_item.dev_path = '/dev/' + src_device.name
-                            break
+                for line in proc_comm.stdout.splitlines():
+                    if 'osencrypt' in line:
+                        majmin = filter(lambda p: re.match(r'\d+:\d+', p), line.split())[0]
+                        src_device = filter(lambda d: d.majmin == majmin, self.get_device_items(None))[0]
+                        crypt_item.dev_path = '/dev/' + src_device.name
+                        break
 
-                rootfs_dev = next((m for m in self.get_mount_items() if m["dest"] == "/"))
-                crypt_item.file_system = rootfs_dev["fs"]
+            rootfs_dev = next((m for m in self.get_mount_items() if m["dest"] == "/"))
+            crypt_item.file_system = rootfs_dev["fs"]
 
-                if not crypt_item.dev_path:
-                    raise Exception("Could not locate block device for rootfs")
+            if not crypt_item.dev_path:
+                raise Exception("Could not locate block device for rootfs")
 
-                crypt_item.luks_header_path = "/boot/luks/osluksheader"
+            crypt_item.luks_header_path = "/boot/luks/osluksheader"
 
-                if not os.path.exists(crypt_item.luks_header_path):
-                    crypt_item.luks_header_path = crypt_item.dev_path
+            if not os.path.exists(crypt_item.luks_header_path):
+                crypt_item.luks_header_path = crypt_item.dev_path
```

Comment:
without the osluksheader how are you updating the header path to dev path? Is this for in-place headers?

Reply:
I am just fixing indentation for some pre-existing code here- so I'm not sure what's going on here either but I'd also guess the same. This is probably assuming that if the header is detached and not located at osluksheader then its attached.